client/state/themes/test/selectors.js

```diff
@@ -985,61 +986,71 @@ describe( 'themes selectors', () => {
 			expect( activeTheme ).to.be.null;
 		} );
 
-		it( 'given a wpcom site, should return its currently active theme', () => {
+		it( 'given a site, should return its currently active theme', () => {
 			const activeTheme = getActiveTheme(
 				{
-					sites: {
-						items: {
-							2916284: {
-								ID: 2916284,
-								options: {
-									theme_slug: 'premium/mood'
-								}
-							}
+					themes: {
+						activeThemes: {
+							2916284: 'twentysixteen'
 						}
 					}
-				}, 2916284
+				},
+				2916284
 			);
 
-			expect( activeTheme ).to.equal( 'mood' );
+			expect( activeTheme ).to.equal( 'twentysixteen' );
 		} );
+	} );
 
-		it( 'given a Jetpack site, should return its currently active theme', () => {
-			const activeTheme = getActiveTheme(
-				{
-					sites: {
-						items: {
-							77203074: {
-								ID: 77203074,
-								jetpack: true,
-								options: {
-									theme_slug: 'twentysixteen'
-								}
-							}
-						}
+	describe( '#isRequestingActiveTheme', () => {
+		it( 'given empty state, should return false', () => {
+			const activeTheme = isRequestingActiveTheme( {} );
+
+			expect( activeTheme ).to.be.false;
+		} );
+
+		it( 'given no active request, should return false', () => {
+			const requestStatus = isRequestingActiveTheme( {
+				themes: {
+					activeThemeRequests: {
+						2916284: false
 					}
-				}, 77203074
-			);
+				}
+			},
+			2916284
+		);
 
-			expect( activeTheme ).to.equal( 'twentysixteen' );
+			expect( requestStatus ).to.be.false;
+		} );
+
+		it( 'given pending action request, should return true', () => {
+			const activationStatus = isRequestingActiveTheme( {
+				themes: {
+					activeThemes: {
+						2916284: 'twentysixteen'
+					},
```

Comment:
No. I was using it when I thought that selector should work differently. Forgot to remove it after I have changed the selector.